package.json

```diff
@@ -11,7 +11,7 @@
 		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
 		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch",
 		"test:unit": "vitest",
-		"lint": "eslint --ext .js,.cjs,.ts,.svelte .",
+		"lint": "eslint --ext .js,.cjs,.ts,.svelte . && tsc --noEmit",
```

Comment:
Let's put this in a separate task called "typecheck"

Reply:
done